src/main/java/org/isf/patient/service/PatientIoOperations.java

```diff
@@ -197,8 +199,8 @@ public boolean deletePatient(Patient patient) throws OHServiceException {
 	}
 
 	/**
-	 * Method that check if a Patient is already present in the DB by his/her name
-	 * (the passed string 'name' should be a concatenation of firstName + " " + secondName
+	 * Method that check if a Patient is already present in the DB by his/her name (the passed string 'name' should be a concatenation of firstName + " " +
+	 * secondName
```

Comment:
no need to reformat all javadoc, especially those with a specific layout

Reply:
corrected